src/services/utilServices/MailClient.ts

```diff
@@ -43,14 +43,38 @@ class MailClient {
     subject: string,
     body: string,
     attachments?: string[]
+  ): Promise<void> {
+    await this.sendMailWithCc(
+      recipient,
+      "IsomerCMS",
+      [],
+      "noreply@isomer.gov.sg",
+      subject,
+      body,
+      attachments
+    )
+  }
+
+  async sendMailWithCc(
+    recipient: string,
+    senderName: string,
```

Comment:
micro nit:feels weird to expose this api only to change the copy. Change all the copy to isomer team? 
or type it to isomerCMS | Isomer Team?

Reply:
Hmm I think this one is a little specific to use Isomer Team? The OTP email should be IsomerCMS since it is not really by the team.

Opting to keep the typing simple for now.